src/Management/src/Endpoint/EndpointOptionsExtensions.cs

```diff
@@ -62,7 +62,14 @@ public static string GetPathMatchPattern(this EndpointOptions endpointOptions, M
             path += '/';
         }
 
-        path += endpointOptions.Path;
+        if (baseRequestPath == ConfigureManagementOptions.DefaultCloudFoundryPath && endpointOptions.Id == "httpexchanges")
```

Comment:
Can this reuse `PermissionsProvider.IsCloudFoundryRequest`?

Reply:
I do not see any more references to "httptrace" in the codebase. What's the intended endpoint path when running on Cloud Foundry?

Reply:
Apps Manager needs to be updated to support /httpexchanges as Spring also provides no support for /trace or /httptrace as of Boot 3, which is now 2 years old